apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -287,6 +335,25 @@ export const FileBrowser = React.memo(() => {
     [renameFile, project.files]
   );
 
+  const getDuplicateFilenameError = (
```

Comment:
Could we document that this is intended to return `null` or a message string? Initially I wasn't sure if it was an oversight that we didn't always re-assign `message` before returning.

Reply:
yep I'll add a comment here